src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3556,9 +3556,7 @@ void ApplicationManagerImpl::ForbidStreaming(
 }
 
 void ApplicationManagerImpl::OnAppStreaming(
-    uint32_t app_id,
-    protocol_handler::ServiceType service_type,
-    const Application::StreamingState new_state) {
```

Comment:
I think the old OnAppStreaming should be marked deprecated with functionality left as is. A new OnAppStreaming method can be created with the new signature/functionality. 

This is to prevent a major version change for Core since OnAppStreaming is a public method in application manager.

Reply:
Done in 3d6a459